pytest_tests/testsuites/object/test_object_lock.py

```diff
@@ -684,3 +752,45 @@ def test_link_object_of_complex_object_should_also_be_protected_from_deletion(
                     self.shell,
                     self.cluster.default_rpc_endpoint,
                 )
+
+    @allure.title("Locked object should be protected from deletion after the storage nodes are restarted")
+    @pytest.mark.parametrize(
+        # Only complex objects are required for this test
+        "new_locked_storage_object",
+        [pytest.lazy_fixture("complex_object_size")],
+        indirect=True,
+    )
+    def test_the_object_lock_should_be_kept_after_restarting_the_nodes(
```

Comment:
it is not about the lock objects mainly, it is about relations b/w the objects: e.g. lock**ed** object should be still locked. both the lock and the lock**ed** object are likely to be in the storage but a relationship b/w them may be lost

Reply:
the comment is just about the test naming

Reply:
> it is not about the lock objects mainly, it is about relations b/w the objects: e.g. lock**ed** object should be still locked. both the lock and the lock**ed** object are likely to be in the storage but a relationship b/w them may be lost

Fixed